hack/windows/Prepare-Node.ps1

```diff
@@ -111,15 +103,8 @@ New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C
 $StartKubeletFileContent = '$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
 $global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")'+ [Environment]::NewLine
 
-if ($ContainerRuntime -eq "docker") {
-    $StartKubeletFileContent +=[Environment]::NewLine +'$netId = docker network ls -f name=host --format "{{ .ID }}"
-
-if ($netId.Length -lt 1) {
-    docker network create -d nat host
-}' + [Environment]::NewLine
-}
-
 if ($InstallKubeProxy) {
+    Write-Host "Running Antrea with kube-proxy is no longer supported, this parameter will be removed soon"
     $StartKubeletFileContent += [Environment]::NewLine + '& C:\k\Prepare-ServiceInterface.ps1 -InterfaceAlias "HNS Internal NIC"' + [Environment]::NewLine
```

Comment:
Remove this line? I think we can just print the error message for InstallKubeProxy.

Reply:
I wanted to wait for a couple more releases before removing kube-proxy support from the scripts completely (see PR description). Do you think we should just remove them now?
I personally think that would be acceptable, let's see if anyone else has an opinion on that.

Reply:
@wenyingd @luolanzone let me know what you think
We can either keep the current version, or drop KubeProxy support completely from the scripts

Reply:
Both works for me, if it's removed from K8s 1.26, I guess it should be safe to remove as well.

Reply:
Let's keep it as is since there is no real drawback and we are close to the release date. I will remove it post v2 and we can have the change in v2.1. Sounds acceptable?